src/o365/spo/commands/list/list-view-list.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo } from '../../spo';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+}
+
+class SpoListViewListCommand extends SpoCommand {
+  public allowUnknownOptions(): boolean | undefined {
+    return true;
+  }
+
+  public get name(): string {
+    return commands.LIST_VIEW_LIST;
+  }
+
+  public get description(): string {
+    return 'Gets existing list view';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.listId = typeof args.options.listId !== 'undefined';
+    telemetryProps.listTitle = typeof args.options.listTitle !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const resource: string = Auth.getResourceFromUrl(args.options.webUrl);
+    let siteAccessToken: string = '';
+    const baseRestUrl: string = `${args.options.webUrl}/_api/web/lists`;
+    const listRestUrl: string = args.options.listId ?
+      `(guid'${encodeURIComponent(args.options.listId)}')`
+      : `/getByTitle('${encodeURIComponent(args.options.listTitle as string)}')`;
+    const viewRestUrl: string = `/views`;
+
+    if (this.debug) {
+      cmd.log(`Retrieving access token for ${resource}...`);
+    }
+
+    auth
+      .getAccessToken(resource, auth.service.refreshToken as string, cmd, this.debug)
+      .then((accessToken: string): request.RequestPromise => {
+        siteAccessToken = accessToken;
+
+        if (this.debug) {
+          cmd.log(`Retrieved access token ${accessToken}. Retrieving request digest...`);
+        }
+
+        return this.getRequestDigestForSite(args.options.webUrl, siteAccessToken, cmd, this.debug);
+      })
+      .then((res: ContextInfo): request.RequestPromise => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const requestOptions: any = {
+          url: `${baseRestUrl}${listRestUrl}${viewRestUrl}`,
+          method: 'GET',
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${siteAccessToken}`,
+            'accept': 'application/json;odata=nometadata'
+          }),
+          json: true
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.get(requestOptions);
+      })
+      .then((rawRes: any): void => {
+        // request doesn't return any content
+
+        if (this.verbose) {
```

Comment:
You should replace DONE with the actual raw response you've got from the server. Otherwise the list view list will never be displayed

Replace: 
```
if (this.verbose) {
  cmd.log(vorpal.chalk.green('DONE'));
}
```

with:
```
cmd.log(rawRes);
```

We use `cmd.log(vorpal.chalk.green('DONE'));` in a case of void commands that are not suppose to return anything